lib/src/components/Accordion/tests/index.test.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react'
-import { act, renderHook, screen } from '@testing-library/react'
+import { renderHook, screen } from '@testing-library/react'
+import { describe, expect, it } from 'vitest'
```

Comment:
Ça serait bien de set up `describe`, `it`, `expect` dans vitest.config.ts, non (pour éviter importer dans chaque fichier test) ? 

https://vitest.dev/config/#globals

Reply:
Effectivement je l'avais pas mis au bon endroit, je change 🔥